storm-dist/binary/src/main/assembly/binary.xml

```diff
@@ -340,10 +340,10 @@
         </fileSet>
         <!-- $STORM_HOME/extlib -->
         <fileSet>
-            <directory>${project.basedir}/../../external/storm-kafka-monitor/target</directory>
+            <directory></directory>
```

Comment:
It's needed to create empty directory "extlib". 
https://github.com/apache/storm/blob/1.0.x-branch/storm-dist/binary/src/main/assembly/binary.xml#L335